article_store/README.md

```diff
@@ -0,0 +1,85 @@
+# Article Store
+
+An XML data store for the Libero Walking skeleton.
+
+## Endpoints
+`GET`
+
+`/articles`
+
+Returns list of article ids.
+
+```{xml}
+<libero:articles xmlns:libero="http://libero.pub">
+    <libero:article>0065-1737-azm-31-03-0367</libero:article>
+    <libero:article>0034-1234-azm-32-04-0777</libero:article>
+    <libero:article>0067-1777-azm-34-08-0237</libero:article>
+</libero:articles>
+```
+
+---------------------
+
+`DELETE`
+
+`/articles/<article_id>`
+
+Deletes an entire article including all versions and content.
+
+---------------------
+
+`POST` `PUT` `DELETE`
+
+`/articles/<article_id>/<version>`
+
+`POST` Create an article version, can include content items.
```

Comment:
an example `curl` would be useful here, I tried:
```
curl -v -d '<root>...' http://localhost:8000/articles/0065/1 -X POST
```
but got
```
Exception Type: TypeError at /articles/0065/1
Exception Value: post() got an unexpected keyword argument 'version'
```

Reply:
OK will put some example urls along with the data in the examples.

You can't specify a version in the POST as it just creates the next version for you based on existing versions. You can specify the version using PUT (if it exists) as this will modify an existsing version.